docs/source/getting-started/installation.rst

```diff
@@ -23,6 +23,7 @@ Install SkyPilot using pip:
 
           # Choose your cloud:
 
+          pip install "skypilot[kubernetes]"
```

Comment:
We should mention `socat`, `netcat`, and `kubectl` installation in the detailed setup section for kubernetes. Also, we should double check if they are included in `check_credentials`

Reply:
https://github.com/skypilot-org/skypilot/pull/4807 will handle that.